src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt

```diff
@@ -534,13 +578,19 @@ class PostgresEntityDataQueryService(
      * @param entityKeyIds The entity key ids to tombstone.
      * @param authorizedPropertyTypes The property types the user is allowed to tombstone. We assume that authorization
      * checks are enforced at a higher level and that this just streamlines issuing the necessary queries.
+     * @param partitionsInfo Contains the partition information for the requested entity set.
      */
     fun clearEntities(
-            entitySetId: UUID, entityKeyIds: Set<UUID>, authorizedPropertyTypes: Map<UUID, PropertyType>
+            entitySetId: UUID,
+            entityKeyIds: Set<UUID>,
+            authorizedPropertyTypes: Map<UUID, PropertyType>,
+            partitionsInfo: PartitionsInfo = partitionManager.getEntitySetPartitionsInfo(entitySetId)
     ): WriteEvent {
+        val tombstoneVersion = -System.currentTimeMillis()
```

Comment:
I'm confused. I was liking the pattern of `val version = System.currentTimMillis()` and then using it as `-version` but now this is the other way around. What's the rationale here?

Reply:
Yeah, probably it's more readable that way. I was using the same name `tombstoneVersion` here, just put it 1 call up, because both of `tombstone` functions are using the -version. Will change to -version then.